### PR TITLE
Add apollo tests for unwedge

### PR DESCRIPTION
**Pull request by nkgeorgiev**
Add tests for:
state transfer after unwedge
unwedge with n-f replicas
wedge status command

### CODE REVIEW

File: tests/apollo/test_skvbc_reconfiguration.py [hunk 1]
```diff
@@ -566,6 +566,209 @@ async def test_wedge_command_and_specific_replica_info(self, bft_network):
...
+        bft_network.stop_replicas(replicas_to_stop)
+
+        with log.start_action(action_type="send_unwedge_cmd",
+                              checkpoint_before=checkpoint_before,
+                              stopped_replicas=list(replicas_to_stop)):
+            op = operator.Operator(
+                bft_network.config, client,  bft_network.builddir)
+            await op.unwedge(bft=True)
+
+        with trio.fail_after(seconds=90):
```
Comment (yontyon) [inline tests/apollo/test_skvbc_reconfiguration.py:615]: Use https://github.com/vmware/concord-bft/blob/c54bc03d1783d8734fdfef2a449528e0da10b961/tests/apollo/test_skvbc_reconfiguration.py#L1718 instead

File: tests/apollo/test_skvbc_reconfiguration.py [hunk 2]
```diff
@@ -566,6 +566,209 @@ async def test_wedge_command_and_specific_replica_info(self, bft_network):
...
+        # We increase the default request timeout because we need to have around 300 consensuses which occasionally may take more than 5 seconds
+        client.config._replace(req_timeout_milli=10000)
+        with log.start_action(action_type="send_wedge_cmd",
+                              checkpoint_before=checkpoint_before,
+                              stopped_replicas=list(replicas_to_stop)):
+            op = operator.Operator(
+                bft_network.config, client,  bft_network.builddir)
+            await op.wedge()
+
+        with trio.fail_after(seconds=60):
```
Comment (yontyon) [inline tests/apollo/test_skvbc_reconfiguration.py:678]: same

File: tests/apollo/test_skvbc_reconfiguration.py [hunk 3]
```diff
@@ -566,6 +566,209 @@ async def test_wedge_command_and_specific_replica_info(self, bft_network):
...
+        bft_network.stop_replicas(replicas_to_stop)
+
+        with log.start_action(action_type="send_unwedge_cmd",
+                              checkpoint_before=checkpoint_before,
+                              stopped_replicas=list(replicas_to_stop)):
+            op = operator.Operator(
+                bft_network.config, client,  bft_network.builddir)
+            await op.unwedge(bft=True)
+
+        with trio.fail_after(seconds=90):
```
Comment (yontyon) [inline tests/apollo/test_skvbc_reconfiguration.py:700]: Here we can also use the method from above (but it needed to be changed a bit)

File: tests/apollo/test_skvbc_reconfiguration.py [hunk 4]
```diff
@@ -566,6 +566,209 @@ async def test_wedge_command_and_specific_replica_info(self, bft_network):
...
+        skvbc = kvbc.SimpleKVBCProtocol(bft_network)
+        client = bft_network.random_client()
+        # We increase the default request timeout because we need to have around 300 consensuses which occasionally may take more than 5 seconds
+        client.config._replace(req_timeout_milli=10000)
+
+        op = operator.Operator(
+            bft_network.config, client,  bft_network.builddir)
+        await op.wedge()
+
+        with trio.fail_after(seconds=90):
```
Comment (yontyon) [inline tests/apollo/test_skvbc_reconfiguration.py:740]: same

File: tests/apollo/test_skvbc_reconfiguration.py [hunk 5]
```diff
@@ -566,6 +566,209 @@ async def test_wedge_command_and_specific_replica_info(self, bft_network):
...
+                        done = False
+                        break
+
+        await self.validate_stop_on_super_stable_checkpoint(bft_network, skvbc)
+
+        op = operator.Operator(
+            bft_network.config, client,  bft_network.builddir)
+        await op.unwedge()
+
+        with trio.fail_after(seconds=90):
```
Comment (yontyon) [inline tests/apollo/test_skvbc_reconfiguration.py:759]: same

**Review by yontyon — CHANGES_REQUESTED** [review]
Overall it looks very good
I have only one small comment about using an already existed method for getting the wedge status
Thanks!

**Review by teoparvanov — APPROVED** [review]
Looks great, thanks!!

**Review by yontyon — APPROVED** [review]
LGTM!
Thanks

**Pull request merged**